### PR TITLE
Add star+ support

### PR DESCRIPTION
**Pull request by danilovicente**
Add support to star+ service.

### CODE REVIEW

File: marathon.user.js [hunk 1]
```diff
@@ -940,6 +945,12 @@ async function initGMC() {
         title: "Uncheck if you don't use Disney+",
         default: true,
       },
+      starplus: {
+        type: "checkbox",
+        label: "Star+",
+        title: "Uncheck if you don't use Star+",
+        default: true,
+      },
```
Comment (aminomancer) [inline marathon.user.js:953]: If the `disneyplus()` site method works for starplus, then a checkbox is not needed. Like `primevideo` is just an alias of `amazon` so you can do the same with `starplus` and `disneyplus`. That will leave more room in the settings page for unique sites in the future.

**Comment by aminomancer** [comment]
Looks good thanks, just one minor issue

File: marathon.user.js (hunk 1 above)
Comment (danilovicente) [reply marathon.user.js:953]: Oh ok, gonna remove.

**Review by aminomancer — APPROVED** [review]
Thanks!

**Pull request merged**